tests/kubectl_kadalu_tests.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash -e
+
+DISK="$1"
+
+# This test assumes a kubernetes setup available with host as minikube
+
+function install_cli_package() {
+    apt install python3-pip
+    python3 -m pip install setuptools
+    cd cli;
+    # install kubectl kadalu
+    echo "0.0.1canary"  > VERSION
+    sudo python3 setup.py install
+    cd ..
+}
+
+function test_install() {
+    sed -i -e 's/imagePullPolicy: Always/imagePullPolicy: IfNotPresent/g' manifests/kadalu-operator.yaml
+
+    kubectl kadalu install --local-yaml manifests/kadalu-operator.yaml
+}
+
+function test_storage_add() {
+    sed -i -e "s/DISK/${DISK}/g" tests/get-minikube-pvc.yaml
+    kubectl create -f tests/get-minikube-pvc.yaml
+
+    sleep 1
+    kubectl kadalu storage-add test-volume3 --type Replica3 --device minikube:/mnt/${DISK}/file3.1 --path minikube:/mnt/${DISK}/dir3.2 --pvc local-pvc
```

Comment:
looks like each of these commands failing should result in failure of the test. Please fix.

Reply:
Done!